### PR TITLE
Discovery API Endpoints

### PR DESCRIPTION
**Pull request by srachakonda**

### CODE REVIEW

File: src/main/scala/com/imaginea/activegrid/core/models/AccountInfo.scala [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+  def fromNeo4jGraph(nodeId: Long): Option[AccountInfo] = {
+    repository.withTx {
+      neo =>
+        val node = repository.getNodeById(nodeId)(neo)
+        Some(new AccountInfo(Some(node.getId),
+          repository.getProperty[String](node, "accountId"),
+          InstanceProvider.toInstanceProvider(repository.getProperty[String](node, "providerType").get),
+          repository.getProperty[String](node, "ownerAlias"),
+          repository.getProperty[String](node, "accessKey").get,
```
Comment (eklavya) [inline src/main/scala/com/imaginea/activegrid/core/models/AccountInfo.scala]: will it always be present?

Comment (srachakonda) [reply src/main/scala/com/imaginea/activegrid/core/models/AccountInfo.scala]: Yeah. As it is mandatory field in save request itself.

Comment (eklavya) [reply src/main/scala/com/imaginea/activegrid/core/models/AccountInfo.scala]: All of them will always be present?

Comment (srachakonda) [reply src/main/scala/com/imaginea/activegrid/core/models/AccountInfo.scala]: Nope will add the case handling

**Comment by srachakonda** [comment]
Raised new PR

**Pull request closed without merging**